### ISSUE TITLE
tool gets stuck

### ISSUE THREAD

**Issue by cordoval** [hashicorp/terraform#4182]
tool gets stuck looking up my home directory?

```
~ ./plan                                                                     Luiss-MacBook-Pro-2 [17:01:57]
2015/12/06 17:01:59 [WARN] Invalid log level: "1". Defaulting to level: TRACE. Valid levels are: [TRACE DEBUG INFO WARN ERROR]
2015/12/06 17:01:59 [INFO] Terraform version: 0.6.7  b7c41aed92c2d6ea841f618b54e400fcbf824845+CHANGES
2015/12/06 17:01:59 [DEBUG] Detected home directory from env var: /Users/cordoval
2015/12/06 17:01:59 [DEBUG] Discovered plugin: atlas = /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provider-atlas
2015/12/06 17:01:59 [DEBUG] Discovered plugin: aws = /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provider-aws
2015/12/06 17:01:59 [DEBUG] Discovered plugin: azure = /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provider-azure
2015/12/06 17:01:59 [DEBUG] Discovered plugin: cloudflare = /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provider-cloudflare
2015/12/06 17:01:59 [DEBUG] Discovered plugin: cloudstack = /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provider-cloudstack
2015/12/06 17:01:59 [DEBUG] Discovered plugin: consul = /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provider-consul
2015/12/06 17:01:59 [DEBUG] Discovered plugin: digitalocean = /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provider-digitalocean
2015/12/06 17:01:59 [DEBUG] Discovered plugin: dme = /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provider-dme
2015/12/06 17:01:59 [DEBUG] Discovered plugin: dnsimple = /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provider-dnsimple
2015/12/06 17:01:59 [DEBUG] Discovered plugin: docker = /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provider-docker
2015/12/06 17:01:59 [DEBUG] Discovered plugin: dyn = /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provider-dyn
2015/12/06 17:01:59 [DEBUG] Discovered plugin: google = /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provider-google
2015/12/06 17:01:59 [DEBUG] Discovered plugin: heroku = /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provider-heroku
2015/12/06 17:01:59 [DEBUG] Discovered plugin: mailgun = /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provider-mailgun
2015/12/06 17:01:59 [DEBUG] Discovered plugin: null = /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provider-null
2015/12/06 17:01:59 [DEBUG] Discovered plugin: openstack = /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provider-openstack
2015/12/06 17:01:59 [DEBUG] Discovered plugin: packet = /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provider-packet
2015/12/06 17:01:59 [DEBUG] Discovered plugin: rundeck = /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provider-rundeck
2015/12/06 17:01:59 [DEBUG] Discovered plugin: template = /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provider-template
2015/12/06 17:01:59 [DEBUG] Discovered plugin: terraform = /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provider-terraform
2015/12/06 17:01:59 [DEBUG] Discovered plugin: tls = /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provider-tls
2015/12/06 17:01:59 [DEBUG] Discovered plugin: vsphere = /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provider-vsphere
2015/12/06 17:01:59 [DEBUG] Discovered plugin: chef = /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provisioner-chef
2015/12/06 17:01:59 [DEBUG] Discovered plugin: file = /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provisioner-file
2015/12/06 17:01:59 [DEBUG] Discovered plugin: local-exec = /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provisioner-local-exec
2015/12/06 17:01:59 [DEBUG] Discovered plugin: remote-exec = /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provisioner-remote-exec
2015/12/06 17:01:59 [DEBUG] Detected home directory from env var: /Users/cordoval
2015/12/06 17:01:59 [DEBUG] Attempting to open CLI config file: /Users/cordoval/.terraformrc
2015/12/06 17:01:59 [DEBUG] File doesn't exist, but doesn't need to. Ignoring.
2015/12/06 17:01:59 [DEBUG] Detected home directory from env var: /Users/cordoval
```

just noticed these processes are running:

```
~ ps aux | grep terra                                                        Luiss-MacBook-Pro-2 [17:08:05]
cordoval        26187   0.0  0.0 573386624    296   ??  S    Wed04PM   0:00.08 /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provisioner-file
cordoval        26186   0.0  0.0 573386396    284   ??  S    Wed04PM   0:00.08 /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provisioner-chef
cordoval        26184   0.0  0.0 573385616    284   ??  S    Wed04PM   0:00.08 /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provisioner-local-exec
cordoval        26183   0.0  0.0 573385940    284   ??  S    Wed04PM   0:00.09 /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provider-template
cordoval        26182   0.0  0.0 573386868    284   ??  S    Wed04PM   0:00.08 /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provisioner-remote-exec
cordoval        10450   0.0  0.0 573386652    284   ??  S    Tue02PM   0:00.13 /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provisioner-chef
cordoval        10449   0.0  0.0 573386624    284   ??  S    Tue02PM   0:00.13 /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provisioner-file
cordoval        10448   0.0  0.0 573401408   4156   ??  S    Tue02PM   0:05.71 /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provider-aws
cordoval        10447   0.0  0.0 573386868    296   ??  S    Tue02PM   0:00.13 /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provisioner-remote-exec
cordoval        10446   0.0  0.0 573385940    284   ??  S    Tue02PM   0:00.14 /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provider-template
cordoval        10445   0.0  0.0 573385616    284   ??  S    Tue02PM   0:00.13 /Users/cordoval/Sites/libs/haimdall/terraform/terraform-provisioner-local-exec
cordoval        39054   0.0  0.0  2434836    776 s000  S+    5:08PM   0:00.00 grep --color=auto --exclude-dir=.bzr --exclude-dir=CVS --exclude-dir=.git --exclude-dir=.hg --exclude-dir=.svn terra
```

even after i kill all those processes and run ./plan, still does not work and stalls

**Comment by cordoval** [comment]
i installed the tools and compiled them myself and it is working again. But i am not using the downloadable tools again.

**Comment by phinze** [comment]
Hi @cordoval, looks like you were running 0.6.7 hit one of the bugs that was fixed in 0.6.8 - if you try 0.6.8 it should be AOK! Of course building from the repo always works as well. :+1:

**Comment by cordoval** [comment]
thanks man, i am learning hopefully fast enough, will catch up with you guys, i am hanging out on IRC. Thanks :+1:
